runtime_config_linux.go

```diff
@@ -200,20 +200,20 @@ type Network struct {
 type Resources struct {
 	// DisableOOMKiller disables the OOM killer for out of memory conditions
 	DisableOOMKiller bool `json:"disableOOMKiller"`
```

Comment:
Yeah, I think this should be pointer, the default false would be a problem when you want to update the resource of a existed container.